google_forms_to_jira/program.py

```diff
@@ -1,59 +1,69 @@
-"""This program polls a Google Form for new responses and creates a Jira issue for each new response.
+"""Create Jira issues based on Google Forms responses.
 
-Workflow:
-1. Trigger: HTTP GET request.
-2. Poll Forms: Poll the Google Form for new responses.
-3. Create Jira Issue: For each new response, create a Jira issue with the response data.
+Atlassian Jira API documentation:
+- https://docs.autokitteh.com/integrations/atlassian/jira/python
+
+Google Forms API documentation:
+- https://docs.autokitteh.com/integrations/google/forms/events
 """
 
-import json
 import os
-import time
 
-import autokitteh
 from autokitteh import google
 from autokitteh.atlassian import jira_client
 
 
-POLL_INTERVAL = os.getenv("POLL_INTERVAL")
+JIRA_PROJECT_KEY = os.getenv("JIRA_PROJECT_KEY")
```

Comment:
Is this the convention we're using?

**Connection-related strings (e.g., "forms_conn"):** If a connection string is only used a few times in the code, we prefer to rewrite the string directly rather than define a constant. This avoids cluttering the code with unnecessary constants for values that aren't reused extensively.

**Everything else**: For other values that are used frequently or could benefit from a single source of truth (e.g., form IDs, API keys, or frequently referenced text), we use constants. This ensures consistency, reduces the risk of typos, and makes the code easier to update when the value needs to change."

Reply:
Yes. I'd also say that when a const value is defined using some action (e.g. reading an env var, initializing an object) then it makes more sense to do it only in one place.